packages/fxa-auth-server/lib/account-delete.ts

```diff
@@ -11,11 +13,23 @@ import { PayPalHelper } from './payments/paypal/helper';
 import { StripeHelper } from './payments/stripe';
 import push from './push';
 import pushboxApi from './pushbox';
-import { AuthLogger } from './types';
+import { AppConfig, AuthLogger } from './types';
+/*
+import {
+  uid as uidValidator,
+  customerId as customerIdValidator,
+} from './routes/validators';
+//*/
+import { accountDeleteCloudTaskPath } from './routes/cloud-tasks';
+
+export const AccountDeleteReasons = [
+  'fxa_unverified_account_delete',
+  'fxa_user_requested_account_delete',
```

Comment:
'fxa' in these strings because they are going into Stripe so we want that context in the value.